ursgal/uparams.py

```diff
@@ -1,4 +1,137 @@
 ursgal_params = {
+    'reporter_ion_tolerance': {
+        'edit_version': 1.00,
+        'available_in_unode' : [
+            'TMT_quant_1_0_0',
+        ],
+        'triggers_rerun' : True,
+        'ukey_translation' : {
+            'TMT_quant_style_1': 'reporter_ion_tolerance',
+        },
+        'utag' : [
+            'reporter',
+            'accuracy'
+        ],
+        'uvalue_translation' : {
+        },
+        'uvalue_type' : 'float',
+        'uvalue_option' : {
+            'none_val'  : None,
+            'max'       : 10000000,
+            'min'       : 0,
+            'updownval' : 1,
+            'unit'      : '',
+            'f-point'   : 0.2
+        },
+        'default_value' : 0.002,
+        'description' : \
+            'Lower reporter mass tolerance; maximum negative deviation of measured from calculated reporter ion mass.',
+    },
+    'reporter_ion_tolerance_unit': {
+        'edit_version' : 1.01,
+        'available_in_unode' : [
+            'TMT_quant_1_0_0',
+        ],
+        'triggers_rerun' : True,
+        'ukey_translation' : {
+            'TMT_quant_style_1': 'reporter_ion_tolerance_unit',
+        },
+        'utag' : [
+            'reporter',
+            'accuracy'
+        ],
+        'uvalue_translation' : {
+        },
+        'uvalue_type' : 'select',
+        'uvalue_option' : {
+            'select_type' : 'radio_button',
+            'available_values'  : ['da', 'mmu', 'ppm'],
+            'custom_val_max' : 0,
+        },
+        'default_value' : 'da',
+        'description' : \
+            'reporter mass tolerance unit: available in ppm '\
+            '(parts-per-millon), da (Dalton) or mmu (Milli mass unit)',
+    },
+    'impurity_matrix': {
+        'edit_version' : 1.01,
+        'available_in_unode' : [
+            'TMT_quant_1_0_0',
+        ],
+        'triggers_rerun' : True,
+        'ukey_translation' : {
+            'TMT_quant_style_1': 'impurity_matrix',
+        },
+        'utag' : [
+            'reporter',
+            'accuracy'
+        ],
+        'uvalue_translation' : {
+        },
+        'uvalue_type' : 'select',
```

Comment:
wrong uvalue_type and uvalue_option, isn't it?

Reply:
Is uvalue_type still used?

Reply:
I think it is not used but i would keep it, can never be wrong to have the correct type of the value

Reply:
uvalue_type will be kept (at least that's my memory) since it's actually useful, uvalue_option will be removed, that's true, so don't worry about that one too much.